src/promptflow/tests/test_configs/flows/flow_with_non_english_input/flow.dag.yaml

```diff
@@ -0,0 +1,17 @@
+$schema: https://azuremlschemas.azureedge.net/promptflow/latest/Flow.schema.json
```

Comment:
what's the error if the flow.dag.yaml itself is not encoded as utf-8 and has multi-byte character in it?

Reply:
If the yaml itself is not encoded as utf-8 and has multi-byte character, then the input in yaml would be like this:
![image](https://github.com/microsoft/promptflow/assets/2572521/4f4f430d-913b-4b95-aaae-dae59329d4ee)

But for promptflow, we load it without error because we think it as a string and the execution result would be:
![image](https://github.com/microsoft/promptflow/assets/2572521/3301409c-81fa-4ea5-a707-351094b10d53)